resources/options_lookup.tsv

```diff
@@ -541,7 +541,17 @@ Windows	"Low-E, Double, Low-Gain"	ResidentialConstructionsWindowsSkylights	windo
+Windows	"Low-E Storm on Double, Non-metal"	ResidentialConstructionsWindowsSkylights	window_ufactor=0.29	window_shgc=0.42	skylight_ufactor=0.37	skylight_shgc=0.3		Specs Source																																							
+Windows	"Double, Clear, Metal, Air"	ResidentialConstructionsWindowsSkylights	window_ufactor=0.76	window_shgc=0.67	skylight_ufactor=0.37	skylight_shgc=0.3		copied from differently-worded equivalent option above																																							
+Windows	"Double, Clear, Metal, Air, Exterior Clear Storm"	ResidentialConstructionsWindowsSkylights	window_ufactor=0.55	window_shgc=0.51	skylight_ufactor=0.37	skylight_shgc=0.3		"Culp et al July 2015, Table 2, double hung, worst-case mounting"																																							
+Windows	"Double, Clear, Non-metal, Air"	ResidentialConstructionsWindowsSkylights	window_ufactor=0.49	window_shgc=0.56	skylight_ufactor=0.37	skylight_shgc=0.3		copied from differently-worded equivalent option above																																							
+Windows	"Double, Clear, Non-metal, Air, Exterior Clear Storm"	ResidentialConstructionsWindowsSkylights	window_ufactor=0.34	window_shgc=0.49	skylight_ufactor=0.37	skylight_shgc=0.3		"Culp et al July 2015, Table 1 double hung"																																							
+Windows	"Double, Low-E, Non-metal, Air, M-Gain"	ResidentialConstructionsWindowsSkylights	window_ufactor=0.38	window_shgc=0.44	skylight_ufactor=0.37	skylight_shgc=0.3		copied from differently-worded equivalent option above																																							
+Windows	"Single, Clear, Metal"	ResidentialConstructionsWindowsSkylights	window_ufactor=1.16	window_shgc=0.76	skylight_ufactor=0.37	skylight_shgc=0.3		copied from differently-worded equivalent option above																																							
+Windows	"Single, Clear, Metal, Exterior Clear Storm"	ResidentialConstructionsWindowsSkylights	window_ufactor=0.67	window_shgc=0.56	skylight_ufactor=0.37	skylight_shgc=0.3		"Culp et al July 2015, Table 2, double hung, worst-case mounting"																																							
+Windows	"Single, Clear, Non-metal"	ResidentialConstructionsWindowsSkylights	window_ufactor=0.84	window_shgc=0.63	skylight_ufactor=0.37	skylight_shgc=0.3		copied from differently-worded equivalent option above																																							
+Windows	"Single, Clear, Non-metal, Exterior Clear Storm"	ResidentialConstructionsWindowsSkylights	window_ufactor=0.47	window_shgc=0.54	skylight_ufactor=0.37	skylight_shgc=0.3		"Culp et al July 2015, Table 1 double hung"																																							
+Windows	"Triple, Low-E, Non-metal, Air, L-Gain"	ResidentialConstructionsWindowsSkylights	window_ufactor=0.29	window_shgc=0.26	skylight_ufactor=0.37	skylight_shgc=0.3		copied from differently-worded equivalent option above																																							

```

Comment:
Did you verify that having the specs source comments there doesn't break anything? If not, great! Add a '#' so that we know they are comments.

Make sure to retain and rename the ""Low-E, Double, Non-metal, Air, L-Gain"" option too.

Reply:
No, those were just internal notes for while I was working on it. I deleted most of them before committing but it looks like I missed a few.